client/src/legacy/CMSMain.Tree.js

```diff
@@ -140,6 +141,52 @@ $.entwine('ss.tree', function($) {
         }
       };
       return config;
+    },
+
+    /**
+     * Validates the moving of a node
+     * @param data data provided by the move_node.jstree event
+     * @returns {Promise<boolean>} Returning false will prevent the node from moving
+     */
+    canMove: async function(data) {
+      // only display warning if its the homepage
+      var isHomePage = data.rslt.o.find(".homepage").first().length > 0;
+      if (!isHomePage) {
+        return true;
+      }
+
+      // only display warning if we're moving to a new parent
+      var oldParentId = data.rslt.op.data('id');
+      var newParentId = data.rslt.np.data('id');
+      if (oldParentId === newParentId) {
+        return true;
+      }
+
+      var message = i18n.inject(
+        i18n._t(
+          'CMS.RemoveHomePageWarningMessage',
+          'Warning: This page has the URL of {home}. ' +
+          'This is required by your website to  identify the home page. ' +
+          'Without this, visitors will not be able to view your home page.'
+        ),
+        { home: "<em>home</em>" }
+      );
+
+      // split by the home tag so we can inject it to React without escaping the HTML
+      const regex = /(.*)(<em>home<\/em>)(.*)/;
```

Comment:
It seems like overly complex logic just for some italics, to be honest. I'd rather see if @silverstripeux would compromise and say `This page has a URL of "home".` instead. Also, should we be saying "URL segment" rather than "URL", since it's both not a URL and doesn't match the UI label for the field that contains that value?

Reply:
You really need this PR: https://github.com/silverstripe/silverstripe-admin/pull/895

Reply:
😰

Reply:
^^ Yeah using quotations are fine but it should be single quote marks (') not double ("), as used in other parts of the CMS. 

We can use 'URL segment' over 'URL'